script.service.kodi.callbacks/resources/language/English/strings.po

```diff
@@ -17,7 +20,7 @@ msgid "Callbacks for Kodi"
 msgstr ""
 
 msgctxt "Addon Description"
-msgid "Provides user definable actions for specific events within Kodi. Credit to Yesudeep Mangalapilly (gorakhargosh on github) and contributors for watchdog and pathtools modules."
+msgid "Provides user definable actions for specific events within Kodi. Credit to Yesudeep Mangalapilly (gorakhargosh on github) and contributors for watchdog and pathtools modules.    "
```

Comment:
I'll fix on next pull request (if I need to squash before this is 'commitable' it will be on this pull, otherwise next revision).